docs/openapi.yaml

```diff
@@ -1191,8 +1193,8 @@ paths:
       parameters:
         - name: principal
           in: path
-          description: Stacks address or a Contract identifier (e.g. `SP31DA6FTSJX2WGTZ69SFY11BH51NZMB0ZW97B5P0.get-info`)
-          required: true
```

Comment:
This should be kept because the path parameter is required, though, right?

Reply:
Yes, deleted by mistake. Added back.